app/Http/Controllers/Api/Volumes/SimilarityIndicesController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Biigle\Http\Controllers\Api\Volumes;
+
+use Biigle\Http\Controllers\Api\Controller;
+use Biigle\Volume;
+
+class SimilarityIndicesController extends Controller
+{
+    /**
+     * Get all similarity indices of a volume.
+     *
+     * @api {get} volumes/:id/similarityIndices Get similarity indices
+     * @apiGroup Volumes
+     * @apiName VolumeSimilarityIndex
+     * @apiPermission projectMember
+     * @apiDescription Returns a map of image IDs to their similarity indices.
+     *
+     * @apiParam {Number} id The volume ID
+     *
+     * @apiSuccessExample {json} Success response:
+     * {
+     *    "123": 0,
+     *    "321": 1
+     * }
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function index($id)
+    {
+        // should not be find or fail
+        $volume = Volume::findOrFail($id);
```

Comment:
This will automatically return 404 responses for non-image volumes:
```suggestion
        $volume = Volume::where('media_type_id', MediaType::imageId())->findOrFail($id);
```
You also have to add a `use` statement for `Biigle\MediaType`.